Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -534,8 +534,8 @@ void HTMLHyperlinkElementUtils::follow_the_hyperlink(Optional<String> hyperlink_
     // set to source.
     // FIXME: "navigate" means implementing the navigation algorithm here:
     //        https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate
-    hyperlink_element_utils_queue_an_element_task(Task::Source::DOMManipulation, [url_string, &target] {
-        target->loader().load(url_string, FrameLoader::Type::Navigation);
+    hyperlink_element_utils_queue_an_element_task(Task::Source::DOMManipulation, [url_string, target] {
+        const_cast<FrameLoader&>(target->loader()).load(url_string, FrameLoader::Type::Navigation);
```

Comment:
make the lambda `mutable` instead?

Reply:
A `GCPtr` to a const thing would instead be something like `GCPtr<Foo const>`

Reply:
(Anyway, out of scope for this PR)